README.md

```diff
@@ -75,7 +75,9 @@ with presto.dbapi.connect(
 ) as conn:
   cur = conn.cursor()
   cur.execute('INSERT INTO sometable VALUES (1, 2, 3)')
+  cur.fetchall()
   cur.execute('INSERT INTO sometable VALUES (4, 5, 6)')
+  cur.fetchall()
```

Comment:
@ebyhr would it be better to use `.fetchone` here?

Reply:
@findepi I think so.